core/src/main/java/org/eclipse/hono/config/ServiceConfigProperties.java

```diff
@@ -100,6 +101,34 @@ public final void setPort(final int port) {
         }
     }
 
+    /**
+     * Checks if this server requires the usage of a native TLS implementation.
+     * Native TLS implementations offer in general a better performance but may not be available on all platforms.
+     * If true, the server will require the usage of a native TLS implementation.
+     * Server will not start if native implementation is not available on the current system.
+     * If false, the adapter will try to use a native TLS implementation. If no native implementation is available the
+     * default Java platform independent TLS implementation will be used.
+     *
+     * @return {@code true} if the server requires native TLS implementation.
+     */
+    public final boolean isNativeTlsRequired() {
+        return nativeTlsRequired;
+    }
+
+    /**
+     * Sets if this server should require the usage of a native TLS implementation.
+     * Native TLS implementations offer in general a better performance but may not be available on all platforms.
+     * If true, the server will require the usage of a native TLS implementation.
+     * Server will not start if native implementation is not available on the current system.
+     * If false, the adapter will try to use a native TLS implementation. If no native implementation is available the
+     * default Java platform independent TLS implementation will be used.
```

Comment:
please use the `{@code true}` formatting for the literal boolean values and also document the default value.